tests/src/main/java/net/neoforged/neoforge/debug/fluid/FluidTypeTest.java

```diff
@@ -76,7 +78,7 @@ private static BaseFlowingFluid.Properties fluidProperties() {
                 .bucket(TEST_FLUID_BUCKET);
     }
 
-    private static final RegistryObject<FluidType> TEST_FLUID_TYPE = FLUID_TYPES.register("test_fluid", () -> new FluidType(FluidType.Properties.create().supportsBoating(true).canHydrate(true)) {
```

Comment:
Please add this test (for the stalactite dripping feature) to the javadoc's list, for ease of reference in the future.